simtools/applications/db_development_tools/add_new_parameter_from_json_file_to_db.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a new parameter to the sites collection in the DB
+    using a json file as input.
+
+    This application should not be used by anyone but expert users and not often.
+    Therefore, no additional documentation about this applications will be given.
+
+"""
+
+import logging
+
+import simtools.utils.general as gen
+from simtools.configuration import configurator
+from simtools.db import db_handler
+
+
+def main():
+    config = configurator.Configurator(description=("Add a new parameter to the DB."))
+    config.parser.add_argument("--file_name", help="file to be added", required=True)
+    config.parser.add_argument(
+        "--db_collection", help="DB collection file will be added", required=True
```

Comment:
Description should be "DB collection to which to add the parameter".

Reply:
Fixed.